portability-spi-transfer/src/main/java/org/datatransferproject/spi/transfer/types/FailureReasons.java

```diff
@@ -8,8 +8,11 @@ public enum FailureReasons {
   SESSION_INVALIDATED("SESSION_INVALIDATED"),
   UNCONFIRMED_USER("UNCONFIRMED_USER"),
   UPLOAD_ERROR("UPLOAD_ERROR"),
-  USER_CHECKPOINTED("USER_CHECKPOINTED");
-
+  USER_CHECKPOINTED("USER_CHECKPOINTED"),
+  SERVICE_UNAVAILABLE("SERVICE_UNAVAILABLE"),
```

Comment:
Are all these new ones still used? From a quick glance seems like only the DESTINATION_NOT_FOUND one is?

Reply:
Updated ! One further question, do we need to add these IOExceptions into some config files ? 
All IOExceptions thrown from apple extension are expected to be retry and skip at maximum retry.